<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-P540B2CO" failures="3" tests="12" name="Features.TestRunner" time="412.415" errors="0" timestamp="14 Dec 2020 10:45:16 GMT" skipped="0">
  <testcase classname="Features.TestRunner" name="feature" time="42.184">
    <failure message="expected [rgb(170, 73, 53)] but found [rgb(210, 209, 209)]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [rgb(170, 73, 53)] but found [rgb(210, 209, 209)]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Features.StepDefLogin.password_textbox_does_display_any_error_message(StepDefLogin.java:169)
at ✽.password textbox does display any error message(Features/add_emp.feature:65)
]]>
    </failure>
  </testcase> <!-- feature -->
  <testcase classname="Features.TestRunner" name="feature" time="25.474"/>
  <testcase classname="Features.TestRunner" name="feature" time="40.155"/>
  <testcase classname="Features.TestRunner" name="feature" time="26.573"/>
  <testcase classname="Features.TestRunner" name="feature" time="27.437">
    <failure message="expected [rgb(170, 73, 53)] but found [rgb(210, 209, 209)]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [rgb(170, 73, 53)] but found [rgb(210, 209, 209)]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Features.StepDefLogin.password_textbox_does_display_any_error_message(StepDefLogin.java:169)
at ✽.password textbox does display any error message(Features/add_emp.feature:65)
]]>
    </failure>
  </testcase> <!-- feature -->
  <testcase classname="Features.TestRunner" name="feature" time="70.042"/>
  <testcase classname="Features.TestRunner" name="feature" time="24.359"/>
  <testcase classname="Features.TestRunner" name="feature" time="36.088"/>
  <testcase classname="Features.TestRunner" name="feature" time="28.719"/>
  <testcase classname="Features.TestRunner" name="feature" time="31.530">
    <failure message="expected [rgb(170, 73, 53)] but found [rgb(210, 209, 209)]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [rgb(170, 73, 53)] but found [rgb(210, 209, 209)]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Features.StepDefLogin.password_textbox_does_display_any_error_message(StepDefLogin.java:169)
at ✽.password textbox does display any error message(Features/add_emp.feature:65)
]]>
    </failure>
  </testcase> <!-- feature -->
  <testcase classname="Features.TestRunner" name="feature" time="25.170"/>
  <testcase classname="Features.TestRunner" name="feature" time="34.684"/>
</testsuite> <!-- Features.TestRunner -->
